test/test-fstab-generator.sh

```diff
@@ -25,8 +26,10 @@ for f in "$src"/test-*.input; do
         # shellcheck disable=SC2064
         trap "rm -rf '$out'" EXIT INT QUIT PIPE
 
+        exp="${f%.input}.expected"
+
         # shellcheck disable=SC2046
-        if [[ "$f" == *.fstab.input ]]; then
+        if [[ "${f##*/}" =~ \.fstab\.input ]]; then
```

Comment:
SC2046 is apparently "Quote this to prevent word splitting". So maybe the annotation can be removed?